content/recipes/applied-examples/eubopen.md

```diff
@@ -0,0 +1,51 @@
+(fcb-eubopen)=
+# FAIRification of High-Content Screening data using Bioimage Archive
+
+````{panels_fairplus}
+:identifier_text: TODO
+:identifier_link: TODO
+:difficulty_level: 3
+:recipe_type: applied_example
+:reading_time_minutes: 15
+:intended_audience: data_manager  
+:has_executable_code: nope
+:recipe_name: FAIR High-Content Screening data
+```` 
+
+
+## Abstract
+
+You run high-content screens to test the effect of compounds on cells. You want to make your data FAIR. This recipe is for you. 
+
+
+## Recipe
+
+1. Download the Python scripts and the metadata template Excel sheet from [https://doi.org/10.5281/zenodo.6325622](https://doi.org/10.5281/zenodo.6325622).
```

Comment:
verified files exist in that archive

Reply:
add link to github repo:
https://github.com/rgiessmann/data-management-for-high-content-screening